### ISSUE TITLE
Issues/FYI

### ISSUE THREAD

**Issue by JakeDEvans** [RMerl/asuswrt-merlin#1244]
Not exactly an issue, but wanted others to know.

If you add IPv6 to your openvpn tunnels, you will need to additionally add rules to your IPv6 firewall if you want traffic to traverse.

I'm not sure if this is intentional or not.

VPN Expert options for IPv6 Tunnels (replace subnets with your vpn and lan subnets, I used (2) from my /48 HE.net tunnel

```
push "comp-lzo yes"
tun-ipv6
server-ipv6 2001:db8:200::/64
push "route-ipv6 2001:db8:100::/64"
push "route-ipv6 2001:db8:200::/64"

```

**Comment by RMerl** [comment]
IPv6 is not supported by OpenVPN tunnels.

**Comment by JakeDEvans** [comment]
Bullshit it isn't.

**Comment by RMerl** [comment]
Let me rephrase that: the OpenVPN implementation in Asuswrt-Merlin does not support IPv6.  If you want to use IPv6, you have to configure everything yourself.  This is not a usage scenario that I support.

There's no need to be rude.

**Comment by JakeDEvans** [comment]
rude begets rude. it works, all I'm saying.
